packages/ui/src/stories/formkit/Email.stories.ts

```diff
@@ -0,0 +1,26 @@
+import { StoryFn } from '@storybook/vue3'
```

Comment:
Wonder if that input is needed at all.

Could be easily replaced by `type="email"`